src/index.ts

```diff
@@ -1,11 +1,8 @@
+/// <reference path="./typings/semver.d.ts" />
```

Comment:
I had tried the approach you mentioned but it didn't work, I've taken another look at it and found out what the issue was.
I think because the build command was `tsc src/index.ts` it was ignoring the tsconfig and using typescript defaults. I've changed the command to `tsc` and it loads the config and behaves as expected.